recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "snowflake-snowpark-python" %}
-{% set version = "1.28.0" %}
+{% set version = "1.29.0" %}
```

Comment:
I remember bumping versions in main may cause test failure in some regression jenkins.
you'd better monitor the jenkins after the merge

Reply:
Got it, will be careful about it